packages/data/addon/utils/search.ts

```diff
@@ -74,20 +75,25 @@ export default {
    *          record - asset record
    *          relevance - distance between record and search query
    */
-  searchDimensionRecords(records, query, resultLimit, page) {
+  searchDimensionRecords(
+    records: NativeArray<unknown>,
+    query: string,
+    resultLimit: number,
+    page = 1
+  ): { record: Record<string, string>; relevance: number }[] {
     let results = [],
       record;
 
     // Filter, map, and sort records based on how close each record is to the search query
     for (let i = 0; i < records.length; i++) {
-      record = records.objectAt(i);
+      record = records.objectAt(i) as Record<string, string>;
 
       // Determine relevance based on string match weight
       let descriptionMatchWeight = this.getPartialMatchWeight(
-          (record.description || '').toLowerCase(),
+          (record?.description || '').toLowerCase(),
```

Comment:
Look like we only are only considering `id` and `description` fields. We need search against the dimension value plus any `suggestion` fields.

Reply:
@cythrawll here is an example of the suggstionColumns: https://github.com/yavin-dev/framework/blob/master/packages/reports/addon/components/filter-values/dimension-select.ts#L131

Reply:
Since we don't really have access to that in the search utl, changed so it weights all fields returned by the dimension search.